### ISSUE TITLE
Hyper-parameters on CIFAR-10 using 1000 labels and 2000 labels

### ISSUE THREAD

**Issue by semi-supervised-paper** [vikasverma1077/ICT#5]
Hi authors,
The idea of ICT is really interesting, and I want to reproduce the results in the paper. However, the result is really bad when I change the --num_labeled parameter to 100 and 200. For example, if I set num_labeled to 100 and keep other hyper-parameters unchanged, test error on the model with best validation error is 68.83999633789062.
Can you please tell me the hyper-parameters used in CIFAR-10 with 1000 and 2000 labels?
Thank you.

**Comment by vikasverma1077** [comment]
--num_labeled parameter to 100/ 200 means that per class there are only 10 /20 labels. what other baselines numbers you are using to compare this?

For hyperparameters with 1000 and 2000 labels,  all the other hyperparameters should be same as that of with 4000 labels ( these are given in the README), except --mixup_consistency.

For less number of labeled data, the pseudo-labels are not so correct, so it is advisable to reduce the --mixup_consistency. You can try --mixup_consistency 5/10/20/50 .
